tools/run-image.toit

```diff
@@ -57,16 +68,20 @@ main arguments:
     print_ "Usage: run-image <image|directory>"
     exit 1
 
+  existing-uuids/Set := {}
```

Comment:
Maybe this part of the setup should be in a separate method, so we don't keep the new set alive while we're calling `boot`?

Reply:
done.